apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepConfirmation.tsx

```diff
@@ -57,13 +59,13 @@ function StepConfirmation({
     );
   }
   if (error) {
-    // // Edit ethereum transaction nonce error because transaction has been validated
-    // if (error.name === "LedgerAPI4xx" && error.message.includes("nonce too low")) {
-    //   const mainAccount = account ? getMainAccount(account, parentAccount) : null;
-    //   if (mainAccount?.currency?.family === "evm") {
-    //     error = new TransactionHasBeenValidatedError();
-    //   }
-    // }
+    // Edit ethereum transaction nonce error because transaction has been validated
+    if (error.name === "LedgerAPI4xx" && error.message.includes("nonce too low")) {
+      const mainAccount = account ? getMainAccount(account, parentAccount) : null;
+      if (mainAccount?.currency?.family === "evm") {
```

Comment:
Same here :see_no_evil: 

For the record, the same mechanism exists on LLD and LLM of using `generated` files + a `sync-families-dispatch` script, if you want to look at some examples.

Reply:
Not as straightforward to fix.

Furthermore, this comes from the original PR (#2441) and did not seem to be challenged at the time (rightfully or not, I don't know)

cf. https://github.com/LedgerHQ/ledger-live/pull/2441/files#diff-805265fed9cfcb5a2cbfbc7788982568409a0ccf80a87d8706aa9be5cfccfb3cR62-R68